rtsp.go

```diff
@@ -495,7 +500,8 @@ func (rc *rtspCamera) initMJPEG(session *description.Session) error {
 			return
 		}
 
-		rc.latestFrame.Store(&img)
+		// manually set nil for avFramePtr since we don't use a pool for mjpeg frames
+		rc.latestOutput.Store(&imageAndPoolItem{img: img, poolItem: nil})
```

Comment:
If we don't use a pool item for mjpeg frames... why is it in a type called `imageAndPoolItem`?
Can we please model this in a way that is less implicit & makes it clear which image are pool managed & which ones are not.

Reply:
I changed the type name to `decoderOutput`